src/mgmt/srv_target.c

```diff
@@ -494,10 +494,10 @@ recreate_pooltgts()
 }
 
 int
-ds_mgmt_tgt_setup(void)
+ds_mgmt_tgt_init(void)
```

Comment:
I believe we do not need to change anything in the file if we move `dss_module_setup_all` to `ds_mgmt_drpc_set_up`. (Apologies for the repeated comment. Could you elaborate a bit why we still need to split `ds_mgmt_tgt_setup`, please? Perhaps I have overlooked something.)

Reply:
Indeed, if we move the whole dss_module_setup_all() into ds_mgmt_drpc_set_up(), no other changes required for this ds_mgmt_tgt_setup() anymore.